nexus/src/cidata.rs

```diff
@@ -37,8 +37,8 @@ struct MetaData<'a> {
 }
 
 fn build_vfat(meta_data: &[u8], user_data: &[u8]) -> io::Result<Vec<u8>> {
-    let file_sectors =
-        meta_data.len().div_ceil(&512) + user_data.len().div_ceil(&512);
+    let file_sectors = Integer::div_ceil(&meta_data.len(), &512)
+        + Integer::div_ceil(&user_data.len(), &512);
```

Comment:
These are clippy changes related to bumping nightly, right? Nothing to do with Diesel per se?